nx.json

```diff
@@ -56,5 +56,8 @@
         "unitTestRunner": "none"
       }
     }
+  },
+  "affected": {
+    "defaultBase": "origin/master"
```

Comment:
just `master` is better, since the default is `main` without mentioning the remote